src/test/java/service/impl/fromTestFile.csv

```diff
@@ -0,0 +1,10 @@
+type,fruit,quantity
+b,banana,20
+b,apple,100
+s,banana,100
+p,banana,13
+r,apple,10
+p,apple,20
+p,banana,5
+s,banana,50
+p,banana,12
```

Comment:
this file shouldn't be pushed to the remote repo :)

Reply:
What does it mean? Do I need to create a folder for these files?

Reply:
How can it test FileReader and FileWriter without these files?

Reply:
sorry for unclear explanation :) Yes, you should create a separate resources folder and place it there.